CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 cmake_minimum_required(VERSION 3.13)
 
+file(READ "include/mxnet/base.h" ver)
+string(REGEX MATCH "MXNET_MAJOR ([0-9]*)" _ ${ver})
+set(ver_major ${CMAKE_MATCH_1})
+string(REGEX MATCH "MXNET_MINOR ([0-9]*)" _ ${ver})
+set(ver_minor ${CMAKE_MATCH_1})
+string(REGEX MATCH "MXNET_PATCH ([0-9]*)" _ ${ver})
+set(ver_patch ${CMAKE_MATCH_1})
+set(_PROJECT_VERSION "${ver_major}.${ver_minor}.${ver_patch}")
```

Comment:
Why set `_PROJECT_VERSION`?

Instead you may want to use something like `project(mxnet C CXX VERSION "${ver_major}.${ver_minor}.${ver_patch}")` instead of the existing `project(mxnet C CXX)`?

Reply:
because ikd can merge it, is is valid, then I merge it

Reply:
missing "language"

https://cmake.org/cmake/help/latest/command/project.html

noy im not sure if need other changes